x-pack/platform/test/api_integration_deployment_agnostic/apis/streams/doc_counts.ts

```diff
@@ -405,7 +405,9 @@ export default function ({ getService }: DeploymentAgnosticFtrProviderContext) {
       });
     });
 
-    describe('Scalability with many streams', () => {
+    describe('Scalability with many streams', function () {
+      // Skip in serverless due to timeout issues with large number of streams
+      this.tags(['skipServerless']);
```

Comment:
This test might run pretty long, not sure whether we should run it as part of CI. I think this kind of thing is better suited for a performance journey - we can tackle it later.